src/mempool/mempool_test.rs

```diff
@@ -97,4 +122,113 @@ mod tests {
         }));
         assert!(!valid)
     }
+
+    const HOST_FOR_TEST: &str = "127.0.0.1";
+    const PORT_FOR_TEST: u32 = 9388;
+
+    fn setup_gossip_mempool(
```

Comment:
Why are there 2 different setup functions? Do the existing tests break with the gossip set up?

Reply:
I think it makes sense to set up the full mempool with gossip for all tests.

Reply:
If you find that the tests are getting out of hand with this approach, let's reconsider though.

Reply:
https://github.com/farcasterxyz/snapchain/pull/200/commits/da9e455b4a4df918e775aec1aaca28ead0881d8b